### ISSUE TITLE
Unable to start function host 2.0.1-beta.37

### ISSUE THREAD

**Issue by rogersmj** [Azure/azure-functions-core-tools#690]
After upgrading (Homebrew on Mac) to 2.0.1-beta.37, I can no longer start the function host. I get this:

```
Azure Functions Core Tools (2.0.1-beta.37)
Function Runtime Version: 2.0.12050.0
: Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager[0]
      User profile is available. Using '/Users/mrogers/.aspnet/DataProtection-Keys' as key repository; keys will not be encrypted at rest.
info: Host.Startup[0]
      Reading host configuration file '/Users/mrogers/Projects/analytic.li-Billing/host.json'
info: Host.Startup[0]
      Host configuration file read:
      {
        "version": "2.0"
      }
fail: Host.General[0]
      A ScriptHost error has occurred
System.DllNotFoundException: Unable to load shared library 'kernel32' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(libkernel32, 1): image not found
```

I can't figure out how to downgrade either. I've tried reinstalling .NET Core 2.1 and this package, and get the same result.

**Comment by vmary2014** [comment]
Getting the same issue if I put the instrumentation key in my local.settings.json, as referred [here].(https://github.com/Azure/azure-functions-host/issues/3364#issuecomment-418295763)

If you have it try to remove it to see if at least you can start it?
BTW I believe I have the latest Functions SDK but it is still occuring.
I'm on Unbuntu 18.04.

**Comment by rogersmj** [comment]
This was resolved when I switched to Node 8.9.4. I had been using 8.6.0.
